clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -1078,14 +1078,11 @@ private void chainFuture(
         }
 
         @Override
-        public String toString() {
-            return "OffsetFetchRequestState{" +
-                    "requestedPartitions=" + requestedPartitions +
-                    ", memberId=" + memberInfo.memberId.orElse("undefined") +
-                    ", memberEpoch=" + (memberInfo.memberEpoch.isPresent() ? memberInfo.memberEpoch.get() : "undefined") +
-                    ", future=" + future +
-                    ", " + toStringBase() +
-                    '}';
+        public String toStringBase() {
+            return super.toStringBase() +
+                    ", memberInfo=" + memberInfo +
```

Comment:
`memberInfo` is from super `RetriableRequestState`. Maybe we should add `toStringBase` to `RetriableRequestState` too?

Reply:
Agree with moving this up to the `RetriableRequestState` maybe, and once there, I would also suggest we simplify the output to show the member id and epoch directly, something like:
```suggestion
                    ", memberId=" + memberInfo.memberId.orElse... +
                    ", memberEpoch=" + memberInfo.memberEpoch +
```
`MemberInfo` is just an internal wrapper to move the 2 together, but when seeing a toString for an OffsetFetch or OffsetCommit request, we just care about `memberId` and `memberEpoch` (they will be included in the request to the broker)

Reply:
I have implemented both suggestions